core/src/main/java/org/apache/carbondata/core/indexstore/AbstractMemoryDMStore.java

```diff
@@ -52,4 +52,13 @@ public UnsafeMemoryDMStore convertToUnsafeDMStore(CarbonRowSchema[] schema)
       throws MemoryException {
     throw new UnsupportedOperationException("Operation not allowed");
   }
+
+  public void setTotalRowCount(long rowCount) {
```

Comment:
Setter methods should not be overridden. You can make rowCount protected in abstract class and avoid overriding the setter method